docs/index.html

```diff
@@ -3,4 +3,4 @@
 <meta http-equiv="refresh" content="0; url=html/index.html" />
 </head>
 <body></body>
-</html>
\ No newline at end of file
+</html>
```

Comment:
I'm not sure why index.html is changing here? Is this related to the typo you mentioned Austin helping you with?